internal/services/users/user_data_source.go

```diff
@@ -122,6 +122,24 @@ func userDataSource() *schema.Resource {
 				Computed:    true,
 			},
 
+			"employee_type": {
+				Description: "Captures enterprise worker type. For example, Employee, Contractor, Consultant, or Vendor.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
+			"cost_center": {
+				Description: "The cost center associated with the user.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
+			"division": {
+				Description: "The name of the division in which the user works.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
```

Comment:
I think these should be Computed rather than Optional? Also could we sort these fields broadly in the following precedence: Required, then Optional, then Computed, and each of those sections alphanumerically?

Reply:
done